resources/language/nl.ini

```diff
@@ -0,0 +1,489 @@
+language.code       = "nl"
+language.direction  = "ltr"
+language.isRTL = "false"
+language.dateformat = "d/m/Y"
+language.jsdateformat = "dd/mm/yy"
+language.timeformat = "h:i A"
```

Comment:
This is the american timeformat with am/pm. For 24h timeformat you can use:
H:i (https://www.php.net/manual/en/function.date.php)

Reply:
24h format looks better indeed.